### PR TITLE
update create-account-auth message

### PR DESCRIPTION
**Pull request by krlosMata**
This PR implements the following:
- update `createAccountAuthorizationSignature` specification
  - implementation could be found in:
    - [hermezjs](https://github.com/hermeznetwork/hermezjs/blob/main/src/hermez-wallet.js#L66) 
    - [rollup contract](https://github.com/hermeznetwork/contracts/blob/master/contracts/hermez/lib/HermezHelpers.sol#L263)

### CODE REVIEW

File: docs/developers/protocol/hermez-protocol/protocol.md [hunk 1]
```diff
@@ -228,15 +228,39 @@ There are two ways to authorize an account creation (that is, an Ethereum addres
...
-`AccountCreationAuthMsg = "I authorize this babyjubjub key for hermez rollup account creation" || compressed-bjj || chainId || hermezAddress`
+where:
+- `chainID`: refers to the ethereum chain identifier
+- `rollupContractAddress`: rollup contract ethereum address
+- `compressed-bjj`: babyjubjub public key in its compressed format represented as hexadecimal string
+
+`signature = eth_signTypedData(domain, types, value)`
 
-`AccountCreationAuthSig = Sign_ecdsa(AccountCreationAuthMsg)`
+Further details on `eth_signTypedData` could be found [here](https://eips.ethereum.org/EIPS/eip-712#specification-of-the-eth_signtypeddata-json-rpc)
```
Comment (jleebee) [inline docs/developers/protocol/hermez-protocol/protocol.md]:
```suggestion
Further details on `eth_signTypedData` can be found [here](https://eips.ethereum.org/EIPS/eip-712#specification-of-the-eth_signtypeddata-json-rpc)
```

**Review by jleebee — CHANGES_REQUESTED** [review]
Just one small change :)

**Review by jesusholoo — DISMISSED** [review]
nice :D

File: docs/developers/protocol/hermez-protocol/protocol.md (hunk 1 above)
Comment (krlosMata) [reply docs/developers/protocol/hermez-protocol/protocol.md]: changed !

**Review by jesusholoo — APPROVED** [review]

**Review by jleebee — APPROVED** [review]

**Pull request merged**